README.md

```diff
@@ -241,6 +242,30 @@ sensu_handler 'tcp_handler' do
   timeout 30
 end
 ```
+### sensu_hook
+Used to define hooks for sensu checks
+#### Properties
+* `command` **required** command to be executed
+* `timeout` **required** duration timeout in seconds (hard stop)
```

Comment:
according to the docs its not required and has a default of `60` seconds. See: https://docs.sensu.io/sensu-go/5.10/reference/hooks/ I also don't see you actually requiring it in the provider so this could be copy/paste.